src/app/Page.js

```diff
@@ -11,15 +13,75 @@ export default class Page extends React.Component<{|+children: Node|}> {
     return (
       <div>
         <header>
-          <nav>
-            <ul style={{listStyle: "none", paddingLeft: 0, margin: 0}}>
+          <nav className={css(style.nav)}>
```

Comment:
Styles like `nav` don’t need to be used by any other modules—and indeed
should not be. Could you define them locally instead of in a public
module? You can just inline them into the `StyleSheet.create` call.

If you have styles that you expect to be used by other modules, then I’m
tentatively okay with those being exposed publicly, but I think that it
might be better practice to just have a dedicated component. For
instance, the `header` style is arguably reusable, but it might be
better to just create a `<Header />` component that encapsulates the
appropriate styles.

Reply:
👍 makes sense to me